LiskHQ/lisk-sdk
# Rename module api to method - Closes #7462

### What was the problem?

This PR resolves #7462

### How was it solved?

- Update name Module API to Module Method

### How was it tested?

- Check there is no `api` or `API` except APIClient
- All build/test should pass
- Check there is no file name with `api`
